README.md

```diff
@@ -46,7 +46,7 @@ $ make gcp
 ```
 
 This will output sane defaults to `.deploy/desired_cluster_profile`.
-Now, edit said file and set your `project-id` and the `gce_ssh_pub_key_file`
+Now, edit said file and set the value `gcp_project` with your `project-id` and add your ssh file path to `gcp_ssh_pub_key_file`
```

Comment:
This doesn't seem to improve clarity to me. What _ssh file_?

Reply:
Well identifying where the project-id value should be entered was my primary concern, it is true the latter comments about the ssh are a bit unclear. Ill try to make that clearer.